packages/query/src/utils/query-generation.ts

```diff
@@ -0,0 +1,43 @@
+import { gql, type TypedDocumentNode } from '@apollo/client';
+import { print, type DocumentNode } from 'graphql';
+import { getFragmentName } from './get-fragment-name';
+
+/**
+ * @param {TypedDocumentNode<unknown,unknown>} fragments Fragment on RootQuery
+ * @return query document with all fragments
+ */
+export function generateRootQuery(
+	fragments: TypedDocumentNode< unknown, unknown >[]
+): DocumentNode {
+	return gql`
+        ${ Object.values( fragments ).map( ( fragmentDoc ) =>
+			print( fragmentDoc )
+		) }
+        query getRoot{
+            ${ Object.values( fragments ).map(
+				( fragmentDoc ) => `...${ getFragmentName( fragmentDoc ) }`
+			) }
+        }
+    `;
+}
```

Comment:
Why isn't prettier/eslint formatting this?

Reply:
Nope, mixed tabs/spaces were committed to this file, and are not getting picked up by our local or CI linters.

 I'm assuming it's because it doesnt recognize the `gql` template literally, and _hoping_ it's because you're using Apollo's literal here instead of GraphQL-Codegen's.

![image](https://github.com/user-attachments/assets/152fcc2a-a51a-4825-a745-fd1d9b1130db)


(PS: I recommend a plugin like https://marketplace.visualstudio.com/items?itemName=nhoizey.gremlins, and to make sure that your VSCode _and_ GH diff settings are set to show whitespace differences )

Reply:
Got it. Fixed it manually for now.